app/pages/error.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="error">
+    <h1>{{ title }}</h1>
+    <p>Ups! Nikogo tu nie ma!</p>
```

Comment:
Trzeba dać jakiś poważniejszy komunikat, "Strona nie została znaleziona".

Reply:
Druga rzecz jest taka, że ten błąd to nie zawsze 404.

Reply:
w tym przypadku jest to zawsze 404 (to nie layout/error)